tests/components/pi_hole/test_init.py

```diff
@@ -88,15 +73,16 @@
         assert await async_setup_component(
             hass,
             pi_hole.DOMAIN,
-            {pi_hole.DOMAIN: [{"host": "pi.hole", "name": "Custom"}]},
+            {pi_hole.DOMAIN: [{
+                "host": "pi.hole",
+                "name": "Custom"
+            }]},
         )
 
     await hass.async_block_till_done()
 
-    assert (
-        hass.states.get("sensor.custom_ads_blocked_today").name
-        == "Custom Ads Blocked Today"
-    )
+    assert (hass.states.get("sensor.custom_ads_blocked_today").name ==
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4667984)